src/azure-cli/azure/cli/command_modules/hdinsight/_params.py

```diff
@@ -189,6 +193,30 @@ def load_arguments(self, _):
         c.argument('encryption_in_transit', arg_group='Encryption In Transit', arg_type=get_three_state_flag(),
                    help='Indicates whether enable encryption in transit.')
 
+        # Autoscale Configuration
+        c.argument('autoscale_type', arg_group='Autoscale Configuration', arg_type=get_enum_type(known_autoscale_types),
+                   help='The autoscale type.')
+        c.argument('autoscale_min_workernode_count', type=int,
+                   options_list=['--autoscale-min-workernode-count', '--autoscale-min-count'],
+                   arg_group='Autoscale Configuration',
+                   help='The minimal workernode count for Load-based atuoscale.')
+        c.argument('autoscale_max_workernode_count', type=int,
+                   options_list=['--autoscale-max-workernode-count', '--autoscale-max-count'],
+                   arg_group='Autoscale Configuration',
+                   help='The max workernode count for Load-based atuoscale.')
+        c.argument('timezone', arg_group='Autoscale Configuration', validator=validate_timezone_name,
+                   completer=get_generic_completion_list(AUTOSCALE_TIMEZONES),
+                   help='The timezone for schedule autoscale type. Values from `az hdinsight autoscale list-timezones`')
+        c.argument('days', nargs='+', arg_group='Autoscale Configuration',
+                   completer=get_generic_completion_list(week_days),
+                   help='A space-delimited list of schedule day. Valid days are {}.'.format(', '.join(week_days)))
```

Comment:
> I think we can use `arg_type=get_enum_type(week_days)` to provide valid values to customers.

Hi @MyronFanQiu ,
if we use `arg_type=get_enum_type(week_days)` the help message about allowed values is not ordered by Monday,Tuesday. CLI order it by Alphabet.
![image](https://user-images.githubusercontent.com/12733681/90101582-8fb0e800-dd71-11ea-8e8a-f606915280ce.png)

Reply:
I see. Enum type would also check whether the argument's value is valid or not. What will happen from server side if user inputs a value like `Thurday` or `Weddnesday`? If the error message is easy to identify the root cause, it's ok to use string instead of enum type. Or we can add simple logic to check it in Azure CLI.

Reply:
> I see. Enum type would also check whether the argument's value is valid or not. What will happen from server side if user inputs a value like `Thurday` or `Weddnesday`? If the error message is easy to identify the root cause, it's ok to use string instead of enum type. Or we can add simple logic to check it in Azure CLI.

Added `arg_type=get_enum_type(week_days)`